src/app/work-item/work-item-board/work-item-board.component.html

```diff
@@ -10,7 +10,7 @@
     <section>
       <toolbar-panel
         context="boardview"
-        [wiTypes]="workItemTypes"
+        [wiTypes]="workItemTypesToolbar"
```

Comment:
You can use the same old one and in ngOnChanges of toolbar-panel component you can do the filtering.